### PR TITLE
Display person's birthdate in case/contact/eventParticipant view title #3827

### LINKED ISSUE #3827: Adding the dob to the person details on right top
### Feature Description

The date of birth (plus age) should shown on every page of the person (contact and case).

### Problem Description

Currently the short information, which person is open shows only the name, but not the date of birth:

grafik

### Proposed Change

As there could be many persons with same or similar names it would be nice to see the date of birth and age on one view, not only by opening the person details.

### PR DESCRIPTION
**Pull request by alexandre-gille**
Closes #3827

### CODE REVIEW

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java [hunk 1]
```diff
@@ -1359,8 +1360,25 @@ public VerticalLayout getCaseViewTitleLayout(CaseDataDto caseData) {
...
+		if (StringUtils.isNotBlank(casePersonFullName)) {
+			caseLabelSb.append(casePersonFullName);
+
+			PersonDto casePerson = FacadeProvider.getPersonFacade().getPersonByUuid(caseData.getPerson().getUuid());
+			if (casePerson.getBirthdateDD() != null && casePerson.getBirthdateMM() != null && casePerson.getBirthdateYYYY() != null) {
+				caseLabelSb.append(" (* ")
+					.append(
+						PersonHelper.formatBirthdate(
+							casePerson.getBirthdateDD(),
+							casePerson.getBirthdateDD(),
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java:1374]: This doesn't look right :) there should be MM and YYYY here

File: sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java [hunk 2]
```diff
@@ -806,7 +809,33 @@ public VerticalLayout getContactViewTitleLayout(ContactDto contact) {
 		classificationLabel.addStyleNames(CssStyles.H3, CssStyles.VSPACE_NONE, CssStyles.VSPACE_TOP_NONE);
 		titleLayout.addComponent(classificationLabel);
 
-		Label contactLabel = new Label(contact.toReference().getCaptionAlwaysWithUuid());
+		String shortUuid = DataHelper.getShortUuid(contact.getUuid());
+		String contactPersonFullname = contact.getPerson().getCaption();
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java:813]: Fullname => FullName

File: sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantsController.java [hunk 3]
```diff
@@ -299,8 +300,27 @@ public VerticalLayout getEventParticipantViewTitleLayout(EventParticipantDto eve
...
+
+			if (eventParticipant.getPerson().getBirthdateDD() != null
+				&& eventParticipant.getPerson().getBirthdateMM() != null
+				&& eventParticipant.getPerson().getBirthdateYYYY() != null) {
+
+				eventLabelSb.append(" (* ")
+					.append(
+						PersonHelper.formatBirthdate(
+							eventParticipant.getPerson().getBirthdateDD(),
+							eventParticipant.getPerson().getBirthdateDD(),
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantsController.java:316]: Same as above

**Review by MateStrysewske — CHANGES_REQUESTED** [review]

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java (hunk 1 above)
Comment (alexandre-gille) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java:1374]: Woopsi doopsi !

**Review by MateStrysewske — APPROVED** [review]

**Pull request merged**